### PR TITLE
docs(create-manifest): Add example for manifest json output PE-740

### PR DESCRIPTION
**Pull request by fedellen**
This PR adds README information for the manifest json output update.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -802,6 +802,32 @@ Creating a `.json` file of your manifest links output can be accomplished here w
 ardrive create-manifest -w /path/to/wallet -f "6c312b3e-4778-4a18-8243-f2b346f5e7cb"  | jq '{links}' > links.json
 ```
 
+Similarly, you can output the raw manifest `.json` file that will be uploaded to your ArDrive folder:
```
Comment (arielmelendez) [inline README.md:805]:
```suggestion
If you'd like to preview the contents of your manifest before uploading, you can perform a dry run and do some lightweight post processing to isolate the data:
```

File: README.md [hunk 2]
```diff
@@ -802,6 +802,32 @@ Creating a `.json` file of your manifest links output can be accomplished here w
...
+
+```shell
+ardrive create-manifest -w /path/to/wallet -f "6c312b3e-4778-4a18-8243-f2b346f5e7cb"  | jq '{manifest.manifest}' > manifest.json
```
Comment (arielmelendez) [inline README.md:808]:
```suggestion
ardrive create-manifest -w /path/to/wallet -f "6c312b3e-4778-4a18-8243-f2b346f5e7cb" --dry-run | jq '{manifest.manifest}' > manifest.json
```

**Review by arielmelendez — DISMISSED** [review]

**Review by matibat — DISMISSED** [review]
LGTM

**Review by jdaev — APPROVED** [review]
Looks good 👍🏽

**Pull request merged**